pkg/autoscaler/autoscaler.go

```diff
@@ -91,10 +93,10 @@ func (agg *totalAggregation) aggregate(stat Stat) {
 		current.lameduck(stat.Time)
 	} else {
 		current.aggregate(stat.AverageConcurrentRequests)
-		if stat.PodName == ActivatorPodName {
-			agg.containsActivatorMetrics = true
+		if strings.HasPrefix(stat.PodName, ActivatorPodName) {
```

Comment:
I doubt this is good enough.  I think we also need `stat.Namespace == system.Namespace`

Reply:
Since `stat.Namespace` doesn't exist (and this is a pre-existing problem), feel free to open an issue and `TODO(#1234)` it.